types/react-sketchapp/tsconfig.json

```diff
@@ -1,25 +1,19 @@
 {
     "compilerOptions": {
+        "allowSyntheticDefaultImports": true,
```

Comment:
Also, what change made this change necessary? It also seems to work fine without this locally.
This can cause issues, since consumers of this library may not have the flag enabled (i.e. the compiler option only applies within this package, not to consumers of it).

Reply:
Same here, had to use it for another module.

fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34637/commits/dcceb8d800c34f28c81d0684d73b55d9afcc6634